doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
Synopsis in enc.pod starts with
```
B<openssl enc -ciphername>
```
and is silent about that you can simply use ciphers' names to invoke on enc command. Wouldn't it be appropriate to do same even in dgst.pod, start the synopsis with `dgst -digestname`? I'd say it would...